lib/framebuffer.cc

```diff
@@ -112,6 +112,56 @@ class DirectRowAddressSetter : public RowAddressSetter {
   int last_row_;
 };
 
+class SM5266RowAddressSetter : public RowAddressSetter {
+public:
+  SM5266RowAddressSetter(int double_rows, const HardwareMapping &h)
+    : row_mask_(0), last_row_(-1),
```

Comment:
You want to set `row_mask_` to `h.a | h.b | h.c`, otherwise these bits are not initialized for output.